### ISSUE TITLE
Update tag commands in gametest-beta to use `hasTag` and `addTag` etc.

### ISSUE THREAD

**Issue by m-doescode** [MrDiamond64/Scythe-AntiCheat#10]
With the new gametest beta, new tag functions have been added such as the above.

I have noticed that in the [latest commit](https://github.com/MrDiamond64/Scythe-AntiCheat/commit/6f0c639eb9ad019285e8e4773a953cce19301fb3) that sometimes commands are still used to handle tags.

e.g.
```js
// to lazy to convert this stuff to .hasTag()
    let tags = player.runCommand(`tag "${member}" list`, World.getDimension('overworld')).statusMessage.replace(/§./g, '').match(/(?<=: ).*$/g);
```

Just for completeness, I'm creating the issue to as reminder for the future. If you don't feel like doing it I can help as well.

**Comment by MrDiamond64** [comment]
should be fixed in 33858468c7ba7d589641422e72116b4b1e7c9f29. theres probably more instances of those though.

**Comment by m-doescode** [comment]
> theres probably more instances of those though.
That might be true, but I just cloned the repo and searched for instances of "runCommand" and none of them had anything to do with tags. So we're probably finished.
